test-it/ViaTests/StringScanInputSizeBug.via

```diff
@@ -0,0 +1,92 @@
+// Autogenerated Vireo assembly file
```

Comment:
Here is a picture of the source VI:
![StringScanInputSizeBug](https://user-images.githubusercontent.com/8452083/60217334-1c2cde00-9832-11e9-833b-1ad66cbedb5d.png)

Reply:
Please assert all the output indicators against expected values in writing tests like this.